app/frontend/src/assets/scss/style.scss

```diff
@@ -42,6 +51,14 @@ p {
   line-height: 1.4;
 }
 
+// links
+a{
```

Comment:
Make sure to use VSCode autoformatter to keep spacing consistent. I think there's supposed to be a space between a and { but can't remember SCSS conventions.